test/plugins/test-trace-hapi.ts

```diff
@@ -49,7 +49,7 @@ describe('hapi', function() {
   });
 
   Object.keys(versions).forEach(function(version) {
-    if (version.substring(4) > 10 && semver.satisfies(process.version, '<4')) {
+    if (Number(version.substring(4)) > 10 && semver.satisfies(process.version, '<4')) {
```

Comment:
It might be better to make this a typescript cast instead of a JS one.

Reply:
Typescript casts, AFAIK don't affect the value of the casted object -- so I don't think that would work here.

In `ts-node`, `'1' as number` yields an typecast error.

Reply:
Yup, that makes sense.